docs/secpoll.zone

```diff
@@ -1,4 +1,4 @@
-@       86400   IN  SOA pdns-public-ns1.powerdns.com. pieter\.lexis.powerdns.com. 2021120304 10800 3600 604800 10800
+@       86400   IN  SOA pdns-public-ns1.powerdns.com. pieter\.lexis.powerdns.com. 2021120901 10800 3600 604800 10800
```

Comment:
```suggestion
@       86400   IN  SOA pdns-public-ns1.powerdns.com. pieter\.lexis.powerdns.com. 2021121001 10800 3600 604800 10800
```

Reply:
ah, thanks! I fixed the changelog instead :):